metabase/metabase
# feat(sdk): expose FilterPicker querying component

Closes https://github.com/metabase/metabase/issues/49752

### Description

Exposes the FilterPicker querying component to the SDK, with some CSS resets.

### How to verify

- Render in the `FilterPicker` component in your `InteractiveQuestion`. Make sure to add a `FilterBar` too, so you can see what filters are applied. You can use the [example from Storybook](https://github.com/metabase/metabase/pull/49768/files#diff-5d65becfe7789e892f74796c905e2cdf2f5a25f03acb45616cdf11f16209e5d2) in this PR.
- Try adding filters.
- The filters should be added, and the question should be ran automatically.

### Demo

[Private Loom](https://www.loom.com/share/2664525dcae0488f8b01e37a2b6f1b74?sid=b87d3d7f-089b-4959-8540-7b99e364b3c4)

![CleanShot 2567-11-08 at 22 32 16@2x](https://github.com/user-attachments/assets/f5c46488-bfa7-4652-bdd9-214444a86618)

### Checklist
